src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -1617,4 +1618,77 @@ enum {
     UCT_IB_MLX5_OBJ_TYPE_RESERVED_QPN = 0x002C,
 };
 
+struct uct_ib_mlx5_cqc_bits {
+    uint8_t status[0x4];
+    uint8_t as_notify[0x1];
```

Comment:
since this is a "special" header file, probably better to match fields indentation to the other structs in this file

Reply:
Thanks, good point!

Reply:
seems not fixed for all structs

Reply:
Oh, I think that I've finally got what you mean.

Reply:
Oh, I think that I've finally got what you mean.